src/Symfony/Component/DependencyInjection/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+2.8.9
+-----
+
+* [BC BREAK] fixed inheritance of the `autowire` flag
```

Comment:
The BC break flag has been removed as you and @jakzal seem to agree for treat it as a bug fix only.